content/rs/concepts/memory-performance/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: Memory and performance
+linktitle: Memory and performance
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases: [
+    /rs/concepts/memory-architecture/_index.md,
+    /rs/concepts/memory-architecture/_index/,
+    /rs/administering/designing-production/performance/_index.md,
+    /rs/administering/designing-production/performance/_index./,
+    /rs/concepts/memory-performance/_index.md,
+    /rs/concepts/memory-performance/_index/,
+
+]
+---
+Redis Enterprise Software has multiple mechanisms in its
+architecture to help optimize storage and performance.
+
+## Memory limits
+
+Database memory limits define the maximum size your database can reach across all database replicas and [shards]({{<relref "rs/concepts/terminology.md#redis-instance-shard">}}) on the cluster. This limit includes data values, keys, module data, and overhead for other features. Your memory limit will also determine the number of shards you'll need.
+
+There are a number of factors to consider when sizing your database:
+
+- **dataset size**: you want your limit to be above your dataset size to leave room for overhead.
+- **database throughput**: high throughput needs a higher memory limit.
```

Comment:
That's not that accurate.
The correct logic is higher throughput requires higher number of shards > that provides higher memory.